lib/iris/tests/unit/analysis/cartography/test__get_lon_lat_coords.py

```diff
@@ -0,0 +1,79 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+"""Test function :func:`iris.analysis.cartography._get_lon_lat_coords"""
+
+import pytest
+
+from iris.analysis.cartography import _get_lon_lat_coords as g_lat_lon
+from iris.coords import AuxCoord
+from iris.tests.stock import lat_lon_cube
+
+dim_only_cube = lat_lon_cube()
+lon_dim = dim_only_cube.coord("longitude")
+lat_dim = dim_only_cube.coord("latitude")
```

Comment:
The style of test seems very different to the existing style for most tests. I realise this is probably due to this only now being possible due to using pytest, however there's still some things I'm unused to here which I'd like to make sure are appropriate in the new pytest context.

That is, running code outside of test functions. I'm used to all or most code required for a test being run inside the function for that test. It seems that the pattern of running code outside these functions could possibly yield performance improvements, though perhaps at the cost of robustnesss.